### ISSUE TITLE
aws_elb health_check should not validate if a PATH is present on TCP, SSL

### ISSUE THREAD

**Issue by jleclanche** [hashicorp/terraform#8547]
https://www.terraform.io/docs/providers/aws/r/elb.html

Example:

```hcl
	health_check {
		healthy_threshold = 2
		unhealthy_threshold = 2
		timeout = 3
		target = "TCP:80/"
		interval = 10
	}
```

This passes validation, but fails with the obscure error `Failure configuring health check for ELB: ValidationError: HealthCheck Target must specify a port number`

Documentation says: `TCP, SSL - PORT is required, PATH is not supported`. In that case, PATH should be validated out.

**Comment by kwilczynski** [comment]
Hello @jleclanche ! Thank you for taking the time to bring this up.

I am actually not sure, whether this should be a code fix and/or documentation update? /cc @stack72

**Comment by jleclanche** [comment]
Well documentation is pretty clear about it. This was simply me replacing "HTTP" in an existing target with "TCP", and paying no attention to the trailing slash, thus the error.

I believe this should fail at the `terraform plan` stage, but it doesn't.

**Comment by kwilczynski** [comment]
@jleclanche I see. We can add a validation there, no problem.

**Comment by stack72** [comment]
Closed via #8578
